frontend/src/pages/PersonalPage.jsx

```diff
@@ -96,11 +94,7 @@ function PersonalPage(props) {
                 for (const group of groupResponseJSON.allGroups) {
                     setGroups((arr) => [...arr, group]);
                 }
```

Comment:
Why do you reset the groups array (on line 92)?
Also, it looks like you can simplify this code to ``setGroups(groupResponseJSON.allGroups)``?

Reply:
Fixed it

Reply:
You don't need ``setGroups([])`` anymore

Reply:
Removed it.